celery/backends/couchbase.py

```diff
@@ -39,6 +39,8 @@ class CouchbaseBackend(KeyValueStoreBackend):
     username = None
     password = None
     quiet = False
+    supports_autoexpire = True
```

Comment:
Since expiration time should be in seconds, I think you should also pass `expires_type=int` to the constructor see the [Redis backend](https://github.com/celery/celery/blob/0a78c6a092c46d013d30d1a2719e4cbd65f28c31/celery/backends/redis.py#L157).